server/models/bhima.sql

```diff
@@ -498,12 +498,10 @@ create table `debitor_group` (
   `name`                varchar(100) not null,
   `account_id`          int unsigned not null,
   `location_id`         char(36) not null,
-  `payment_id`          tinyint unsigned not null default '3',
```

Comment:
Also, since you have made several SQL changes, can you provide update files, like @jmcameron to help ease database migration?